test/command/suite/select/function/string/string_slice/nth/options/wrong_options_type.test

```diff
@@ -0,0 +1,10 @@
+plugin_register functions/string
+
+table_create Memos TABLE_HASH_KEY ShortText
+
+load --table Memos
+[
+{"_key": "Groonga"}
+]
+
+select Memos   --output_columns '_key, string_slice(_key, 2, 2, "a")'
```

Comment:
How about `test/command/suite/select/function/string/string_slice/nth/options/invalid/type.test`?

We already use `invalid` directory name such as `test/command/suite/select/drilldown/plain/invalid/`.

Reply:
I fixed it.